### PR TITLE
Alternate row colors

### PR DESCRIPTION
**Pull request by Jxboss911**
Introduces alternating row colors to the course table for improved readability and visual appeal.

### CODE REVIEW

**Review by jepoorbaugh — CHANGES_REQUESTED** [review]
I think you are re-adding all of the prettier files. Could you delete those?

**Review by jepoorbaugh — COMMENTED** [review]
The colors and styling looks good!

**Comment by Jxboss911** [comment]
> I think you are re-adding all of the prettier files. Could you delete those?

Ok i got rid of those files

File: package-lock.json
Comment (Peyton3995) [inline package-lock.json:1]: This should not be here. Remove it from your branch before we merge

**Review by Peyton3995 — CHANGES_REQUESTED** [review]

**Review by jepoorbaugh — DISMISSED** [review]
Styles look good!

File: package-lock.json
Comment (Jxboss911) [reply package-lock.json:1]: Ok i removed this

File: .github/workflows/lint.yml
Comment (Peyton3995) [inline .github/workflows/lint.yml:1]: Please do not include this file in the PR. Do not also make any changes to this file.

Comment (Peyton3995) [reply .github/workflows/lint.yml:1]: Apologies for not including this.

**Review by Peyton3995 — CHANGES_REQUESTED** [review]

**Review by jepoorbaugh — APPROVED** [review]
Looks great!

**Comment by taigercub** [comment]
looks good to me

**Review by Peyton3995 — APPROVED** [review]
Thank you for removing that file. Seems good to go

**Pull request merged**